### PR TITLE
Update cypress to webpack 5 and yarn upgrade

### PR DESCRIPTION
**Pull request by chrisrhymes**
Update dependencies using `yarn upgrade` and upgrade cypress to use webpack 5. 

The news and search pages displayed eslint errors `Error: Must use destructuring page assignment  react/destructuring-assignment` so it has been disabled to allow `yarn build` to run successfully for the time being.

### CODE REVIEW

File: frontend/.eslintrc.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ module.exports = {
     browser: true,
   },
   rules: {
+    'react/destructuring-assignment': 0,
```
Comment (DamienBurkeWN) [inline frontend/.eslintrc.js:24]: A linter rule to enforce an arguably less clear way of writing a piece of code is a linter rule worth disabling!

**Comment by DamienBurkeWN** [comment]
Getting this on `yarn install`:

```
warning Pattern ["@apollo/client@latest"] is trying to unpack in the same destination "/Users/damienburke/Library/Caches/Yarn/v6/npm-@apollo-client-3.5.7-38051c0a414ebf784319fa5131faa19df24fa9ec-integrity/node_modules/@apollo/client" as pattern ["@apollo/client@^3.3.5"]. This could result in non-deterministic behavior, skipping.
```

File: frontend/package.json [hunk 2]
```diff
@@ -16,10 +16,9 @@
     "e2e:headless": "start-server-and-test dev http://localhost:3000 cypress:headless"
   },
   "dependencies": {
-    "@apollo/client": "^3.3.5",
+    "@apollo/client": "~3.5.7",
     "@apollo/federation": "^0.20.4",
     "@apollo/gateway": "^0.21.0",
-    "@apollo/react-hooks": "^4.0.0",
```
Comment (chrisrhymes) [inline frontend/package.json:22]: The migration guide states to remove @apollo/react-hooks as it is in the @apollo/client package now https://www.apollographql.com/docs/react/migrating/apollo-client-3-migration/#apolloreact-hooks

File: frontend/src/pages/[...path]/index.tsx [hunk 3]
```diff
@@ -57,7 +57,9 @@ type DrupalPageProps = {
 };
 
 const DrupalPage = (page: DrupalPageProps): ReactElement => {
-  const { route } = page.data;
+  const {
```
[the remark this answers is not in the document]
Comment (chrisrhymes) [reply frontend/src/pages/[...path]/index.tsx:60]: Yes, I started doing the destructuring as the linter was complaining. Then I disabled the rule as I agree that it is more readable before.

**Pull request merged**